packages/wonder-blocks-dropdown/components/dropdown-launcher.js

```diff
@@ -0,0 +1,244 @@
+// @flow
+
+import * as React from "react";
+import ReactDOM from "react-dom";
+import {StyleSheet} from "aphrodite";
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import Dropdown from "./dropdown.js";
+import type {Item, DropdownItem} from "../util/types.js";
+import ActionItem from "./action-item.js";
+import OptionItem from "./option-item.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /*
+     * The callback function used to open the Dropdown.
+     */
+    children: (openDropdown: () => void) => React.Element<any>,
+
+    opened?: boolean,
```

Comment:
Could you please add a comment explaining this prop?

Reply:
Also, I'm not sure if you are using this prop somewhere inside this component.

Reply:
at the moment I'm not, but its there incase a parent component needs to override the open state.